src/paper/tasks.py

```diff
@@ -1512,13 +1512,13 @@ def celery_create_paper(self, celery_data):
 
         async_paper_updator = getattr(paper_submission, "async_updator", None)
         paper = Paper(**paper_data)
-
         if async_paper_updator is not None:
             paper.doi = async_paper_updator.doi
             paper.hub.add(*async_paper_updator.hubs)
             paper.title = async_paper_updator.title
 
         paper.full_clean()
+        paper.get_pdf_link()
```

Comment:
With the previous suggestion, 
```suggestion
        paper.get_pdf_link(should_save=False)
```
since we are already saving the paper at the end of the method.